src/StorageSync/StorageSync/StorageSync.csproj

```diff
@@ -10,6 +10,13 @@
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
   </PropertyGroup>
 
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.StorageSync" Version="2.8.0-preview" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.5.0" />
```

Comment:
Microsoft.Win32.Registry has dependencies on System.CodeDom and has separate implementations for Unix and Windows - you will need to ensure these dlls are included for correct loading on all platforms

Reply:
We are only supported in windows. And these are platform independent assemblies. In unix, they will throw PlatformNotSupported exception which is expected behavior.

---
In reply to: [255704534](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255704534) [](ancestors = 255704534)